CodeEdit/Features/LSP/LanguageClient/LanguageClient+CallHierarchy.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  LanguageClient+CallHierarchy.swift
+//  CodeEdit
+//
+//  Created by Abe Malla on 2/7/24.
+//
+
+import Foundation
+import LanguageServerProtocol
+
+extension LanguageServer {
```

Comment:
File name and extension do not match. Note that this mismatch is present at all the `LanguageServer` extensions